public/components/register_model/register_model.types.ts

```diff
@@ -6,6 +6,7 @@
 export interface Tag {
   key: string;
   value: string;
```

Comment:
Should we use `string | number` for value type? Prefer to use export type Tag = {key: string} & ({value: number, type: "number"} | {value: string, type: "string"}).

Reply:
The default input value from onChange event of `FieldNumber` is `string`, we would need to convert the value in the component. I'm open to this, do you know what does the api expects?

Reply:
We can change it once API finalized.